### DISCUSSION TITLE
Unable to terminate the data importer background process.

### DISCUSSION THREAD

**Discussion by arulselvan008** [pimcore/pimcore#16082] [Q&A / Support] [answered]
Hi JiaJiaJi,

I have imported the image mapping to the data object using the Data Hub data importer.

Manual command execution: sudo php /var/www/pimcore/bin/console datahub:data-importer:process-queue-parallel --processes=2

We are unable to re-run the manual execution of the data hub data importer in our production system.

![Screenshot 2023-10-11 175828](https://github.com/pimcore/pimcore/assets/106297450/9b0b1bc9-f9e3-4d8a-bf21-c6465bb71a41)

Please help us.
@kingjia90 
@fashxp

**Comment by arulselvan008** [comment]
Hi, 

Can you help us to solve this issue.

**Comment by dvesh3** [comment]
@arulselvan008 please see https://pimcore.com/docs/platform/Data_Importer/Troubleshooting_FAQ#2-error-the-command-is-already-running-while-running-the-process-queue-command the solution is there.

**Comment by arulselvan008** [comment, marked as answer]
Hello Divesh Pahuja,

Entries in the lock keys table are empty, and some entries are available in the bundle_data_hub_data_importer_queue table.
I deleted and retested the import procedure; it is now working fine. Thank you for your response.

Regards,
Arul
